src/page/template/modals.htm

```diff
@@ -13,10 +13,10 @@
     <div class="modal__body" data-bind="fadingscrollbar">
       <div class="modal__text" data-uie-name="status-modal-text">
         <!-- ko if: content().messageHtml -->
-          <div data-bind="html: content().messageHtml"></div>
+          <div id="modal-description" data-bind="html: content().messageHtml"></div>
         <!-- /ko -->
         <!-- ko if: content().messageText -->
-          <div data-bind="text: content().messageText"></div>
+          <div id="modal-description" data-bind="text: content().messageText"></div>
```

Comment:
I can see the `id` twice - are you sure there is always one `div` of these two at a time rendered?

Reply:
one is serving a message and the second one is serving HTML (2 separate if's)

Reply:
👍